example/src/stories/Text.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {RnBaseText} from 'rn-base-component'
+
+export default {
+  title: 'Text',
+  component: RnBaseText,
+} as ComponentMeta<typeof RnBaseText>
+
+export const BasicText: ComponentStory<typeof RnBaseText> = args => (
+  <RnBaseText>
+    {'\n'}
+    <RnBaseText.Bold {...args} />
```

Comment:
Can the code `<RnBaseText.Bold {...args} {'\n'} />` work the same?
it will be more clean

Reply:
it has the wrong syntax and doesn't has props for  `{'\n'}`